docs/features/analysis.md

```diff
@@ -541,6 +541,53 @@ encountered.
           ))
 ```
 
+## Dry-Run Mode
+
+`dryRun` can be used on a metric to control whether or not to evaluate that metric in a dry-run mode. A metric running 
+in the dry-run mode won't impact the final state of the rollout or experiment even if it fails or the evaluation comes 
+out as inconclusive.
+
```

Comment:
We will need a doc stating this is available since v1.2 (since we don't have versioend docs yet)

Reply:
Added, Thanks!